project/ISSUE-TRIAGE.md

```diff
@@ -0,0 +1,109 @@
+## Triaging of Issues
+
+Docker has a lot of issues and it can be sometimes daunting to look through
+everything to find something to work on.  Triaging issues is important to
+help both the author convey the real problem or request that they have and
+to help developers easily find something to work on.
+
+Remember there are more ways than just code that you can contribute to an
+open source project and the community would really thank you if you spent
+spend time triaging issues.
+
+
+### Important first steps
+
+Before jumping into an issue very far make sure that the author of the issue
+has provide enough information so that you are able to make an educated 
+reccomendation on how this is to be triaged.  Standard information that *must*
+be included in most issues are things such as:
+
+* docker version
+* docker info
+* host distro ( ubuntu, RHEL, fedora )
+
+A bonus is to ask for a way to reproduce the issue.
```

Comment:
I'd actually move the "reproduce" part into the *must* section.  W/o a clear way to reproduce it we're just guessing as to what they're seeing.  Whether its just a narrative description of what they were doing, or something as concrete as their Dockerfile, we need something to try to mimic their setup/env/flow.